src/lib/mina_base/zkapp_account.ml

```diff
@@ -389,6 +456,10 @@ let to_input (t : t) : _ Random_oracle.Input.Chunked.t =
     ~zkapp_uri:(f zkapp_uri_to_input)
   |> List.reduce_exn ~f:append
 
+(** Default zkApp account state.
+
+    Used when converting a regular account to a zkApp account. All state
+    is zeroed, no verification key, and proved_state is false. *)
```

Comment:
You might want to clarify that app_state is zeroed and action_state is initialized to the empty action state hash